packages/cli/src/controllers/aiAssistant.controller.ts

```diff
@@ -0,0 +1,32 @@
+import { Post, RestController } from '@/decorators';
+import { BadRequestError } from '@/errors/response-errors/bad-request.error';
+import { AiAssistantService } from '@/services/aiAsisstant.service';
+import { AiAssistantRequest } from '@/requests';
+import { Response } from 'express';
+import type { AiAssistantSDK } from '@n8n_io/ai-assistant-sdk';
+
+@RestController('/ai-assistant')
+export class AiAssistantController {
+	constructor(private readonly aiAssistantService: AiAssistantService) {}
+
+	@Post('/chat')
```

Comment:
do we want to rate-limit this endpoint to make sure that it's not getting abused? 

perhaps something like this to set the limit to 100 requests for every 5 minute window.

```suggestion
	@Post('/chat', { rateLimit: { limit: 100 } })
```